frontend/src/app/[locale]/search/error.tsx

```diff
@@ -31,138 +21,91 @@ interface ErrorProps {
 
 export interface ParsedError {
   message: string;
-  searchInputs: QueryParamData;
+  searchInputs: ServerSideSearchParams;
   status: number;
   type: string;
 }
 
+function isValidJSON(str: string) {
+  try {
+    JSON.parse(str);
+    return true;
+  } catch (e) {
+    return false; // String is not valid JSON
+  }
+}
+
+function createBlankParsedError(): ParsedError {
```

Comment:
it seems like there should be a better way but I think I'd need more context into our error throwing / handling patterns to take an informed approach. For now just tried to change as little as possible to keep this working